cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/schedule/store/DatasetBasedTimeScheduleStore.java

```diff
@@ -104,7 +104,7 @@ public void storeJobAndTrigger(JobDetail newJob, OperableTrigger newTrigger)
     super.storeJob(newJob, true);
     super.storeTrigger(newTrigger, true);
 
-    executePersist(newJob, newTrigger);
+    executePersist(newJob, newTrigger, super.getTriggerState(newTrigger.getKey()));
```

Comment:
Oh I thought you made this comment on some other line (Line: 95 storeJobsAndTriggers method which is not implemented yet and github is wrapping lines). Something similar to what you mentioned here needs to be done to implement that too.
But looks like your comment was on this line itself. Sorry for this misunderstanding. I will change it.